x-pack/plugins/security_solution/common/detection_engine/schemas/request/perform_bulk_action_schema.test.ts

```diff
@@ -8,18 +8,22 @@
 import { performBulkActionSchema, PerformBulkActionSchema } from './perform_bulk_action_schema';
 import { exactCheck, foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
 import { left } from 'fp-ts/lib/Either';
-import { BulkAction } from '../common/schemas';
+import { BulkAction, BulkActionUpdateType } from '../common/schemas';
+
+const retrieveValidationMessage = (payload: PerformBulkActionSchema) => {
```

Comment:
I think `unknown` would be a more accurate type than `PerformBulkActionSchema` as we're testing random payload as input. So such castings as `payload as unknown as PerformBulkActionSchema` could be omitted.

Reply:
This makes sense, thanks for the suggestion